apps/astarte_appengine_api/lib/astarte_appengine_api_web/router.ex

```diff
@@ -19,106 +19,171 @@ defmodule Astarte.AppEngine.APIWeb.Router do
   use Astarte.AppEngine.APIWeb, :router
   alias Astarte.AppEngine.APIWeb.Plug.LogRealm
 
-  pipeline :api do
+  pipeline :auth_api do
```

Comment:
Since we are naming the pipelines according to their scopes, and this pipeline is scoped to a specific realm, let's call the pipeline `:realm_api`
```suggestion
  pipeline :realm_api do
```